src/librustc_metadata/decoder.rs

```diff
@@ -674,37 +710,47 @@ impl<'a, 'tcx> CrateMetadata {
                             if let Some(def) = self.get_def(child_index) {
                                 callback(def::Export {
                                     def: def,
-                                    name: self.item_name(&self.entry(child_index))
+                                    name: self.item_name(&self.entry(child_index)),
                                 });
                             }
                         }
                         continue;
                     }
-                    EntryKind::Impl(_) | EntryKind::DefaultImpl(_) => continue,
+                    EntryKind::Impl(_) |
+                    EntryKind::DefaultImpl(_) => continue,
 
                     _ => {}
                 }
 
                 let def_key = child.def_key.decode(self);
-                if let (Some(def), Some(name)) = (self.get_def(child_index),
-                                                  def_key.disambiguated_data.data.get_opt_name()) {
-                    callback(def::Export { def: def, name: name });
+                if let (Some(def), Some(name)) =
+                    (self.get_def(child_index), def_key.disambiguated_data.data.get_opt_name()) {
+                    callback(def::Export {
+                        def: def,
+                        name: name,
+                    });
                     // For non-reexport structs and variants add their constructors to children.
                     // Reexport lists automatically contain constructors when necessary.
                     match def {
                         Def::Struct(..) => {
                             if let Some(ctor_def_id) = self.get_struct_ctor_def_id(child_index) {
                                 let ctor_kind = self.get_ctor_kind(child_index);
                                 let ctor_def = Def::StructCtor(ctor_def_id, ctor_kind);
-                                callback(def::Export { def: ctor_def, name: name });
+                                callback(def::Export {
+                                    def: ctor_def,
+                                    name: name,
+                                });
```

Comment:
I think this is more readable as it was (I think of the struct's fields as named arguments here).